test/reload-source-on-error.test.js

```diff
@@ -25,6 +29,25 @@ QUnit.module('ReloadSourceOnError', {
       return this.tech;
     };
 
+    this.player.tech_ = () => {
+      return this.tech;
+    };
+
+    this.player.tech_.trigger = (event) => {
```

Comment:
while functionally this works, it doesn't make much sense to use `trigger()` to also get the values of whether other events have been triggered. 

You can mock the `tech` in a similar way the player is so that it gets event handling automatically.

```
this.tech = new videojs.EventTarget();
this.tech.currentSource_ = {
  src: 'thisisasource.m3u8',
  type: 'doesn\'t/matter'
};
this.player.tech = () => this.tech;
this.player.tech_ = this.tech;
```

With this you should be able to listen on the tech for the events you want within the individual test runs